Documentation/Architecture/InputSystem/CoreSystem.md

```diff
@@ -1,6 +1,6 @@
 # Core system
 
-At the heart of the input system is the [MixedRealityInputSystem](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs),
+At the heart of the input system is the [MixedRealityInputSystem](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MRTK.Services/InputSystem/MixedRealityInputSystem.cs),
```

Comment:
Should most of these just be updated to `xref`s?

Reply:
oh, now that is a good thought

Reply:
wonder why they were blob links originally

Reply:
done!